### PR TITLE
add docs/nix.md with some info re. cachix

### PR DESCRIPTION
**Pull request by aryairani**

### CODE REVIEW

File: docs/nix.md
Comment (ceedubs) [inline docs/nix.md:3]: Who is "We"? Maintainers who have been given a cachix token?

Comment (ceedubs) [inline docs/nix.md:3]: Some documentation is better than no documentation, so I wouldn't hold up the PR on this, but it might be nice to have a gentle and concise introduction into why Unison does anything with Nix and what cachix is.

**Review by ceedubs — APPROVED** [review]

File: docs/nix.md
Comment (aryairani) [inline docs/nix.md:3]:
```suggestion
<!-- it might be nice to have a gentle and concise introduction into why Unison does anything with Nix and what cachix is. -->

We can push any nix store path into our cachix cache. This is typically done with `cachix push unison <STORE_PATH>`.
```

File: docs/nix.md
Comment (aryairani) [inline docs/nix.md]:
```suggestion
(Todo: it might be nice to have a gentle and concise introduction into why Unison does anything with Nix and what cachix is.)
```

File: docs/nix.md
Comment (aryairani) [reply docs/nix.md:3]: I added a todo for this

Comment (aryairani) [reply docs/nix.md:3]: I think so, or maybe ["author's we"](https://en.wikipedia.org/wiki/We#Author's_we).

**Pull request merged**
